components/rmrk/Profile/NavbarProfileDropdown.vue

```diff
@@ -293,18 +293,29 @@
       </b-dropdown-item>
 
       <hr class="dropdown-divider mx-4" aria-role="menuitem" />
-
-      <b-dropdown-item
-        custom
-        aria-role="menuitem"
-        paddingless
-        class="is-flex is-justify-content-center">
-        <b-button
-          class="navbar__sign-out-button menu-item mb-4 is-size-7"
-          @click="disconnect()">
-          {{ $t('profileMenu.disconnect') }}
-        </b-button>
-      </b-dropdown-item>
+      <div class="dropdown-btns">
+        <b-dropdown-item
+          custom
+          aria-role="menuitem"
+          paddingless
+          class="is-flex is-justify-content-center">
+          <ConnectWalletButton
+            label="general.change_account"
+            class="navbar__sign-out-button menu-item mb-4 is-size-7"
+            @closeBurgerMenu="closeBurgerMenu" />
+        </b-dropdown-item>
+        <b-dropdown-item
+          custom
+          aria-role="menuitem"
+          paddingless
+          class="is-flex is-justify-content-center">
+          <b-button
+            class="navbar__sign-out-button menu-item mb-4 is-size-7"
+            @click="disconnect()">
+            {{ $t('profileMenu.disconnect') }}
+          </b-button>
+        </b-dropdown-item>
+      </div>
```

Comment:
I would change this code by something more simple

```suggestion
      <b-dropdown-item
        custom
        aria-role="menuitem">
        <div class="buttons is-justify-content-space-between my-2">
          <ConnectWalletButton
          label="general.change_account"
          class="navbar__sign-out-button menu-item is-size-7"
          @closeBurgerMenu="closeBurgerMenu" />
          <b-button
            class="navbar__sign-out-button menu-item is-size-7"
            @click="disconnect()">
            {{ $t('profileMenu.disconnect') }}
          </b-button>
        </div>
      </b-dropdown-item>
    </b-dropdown>
```